tools/incubator.js

```diff
@@ -32,6 +32,19 @@ const Incubator = new Lang.Class({
         return file.get_path();
     },
 
+    _download_file: function (uri) {
+        let source_file = Gio.File.new_for_uri(uri);
```

Comment:
Hmm, maybe this was a bad suggestion on my part. I now get an "Operation not supported" exception thrown when I do this.

Reply:
I was using a flapjack runtime.

Reply:
I can just mimic the same behavior, with GFile.is_native and Soup, no big deal.